File: ProyectoBaseWebApp/app/Tarifa.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Tarifa extends Model
{
    protected $fillable = [
        'tipo_camion','empresa_id', 'origen_madera_id', 'destino_id', 'valor', 'creador_id',
        'modificador_id'
    ];
    protected $appends = [
        'descripcion_tipo_camion'
    ];
    public function origenMadera() {
        return $this->belongsTo('App\OrigenMadera');
    }
    public function getDescripcionTipoCamionAttribute()
    {
        switch ($this->tipo_camion) {
            case 'B':
                return 'Bananero';
            case 'T':
                return 'Trailer';
            default:
                return 'No definido';
        }
    }
    public function destino() {
        return $this->belongsTo('App\Destino');
    }
    public function scopeActive($query)
    {
        return $query->where('estado', 'A');
    }
    public function creador()
    {
        return $this->belongsTo('App\User');
    }
    public function modificador()
    {
        return $this->belongsTo('App\User');
    }
    public function scopeCurrent($query, $empresaId)
    {
        return $query->where('empresa_id', $empresaId);
    }
}

File: CosechasApp/app/src/main/java/com/techtraining/cosechasapp/db/Tarifa.java
package com.techtraining.cosechasapp.db;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Tarifa {
    @PrimaryKey
    public int id;

    @ColumnInfo(name = "destino_id")
    public int destinoId;

    @ColumnInfo(name = "origen_madera_id")
    public int origenMaderaId;

    @ColumnInfo(name = "valor")
    public double valor;

    @ColumnInfo(name = "empresa_id")
    public int empresaId;

    @ColumnInfo(name = "estado")
    public String estado;
}
